strings-csharp/StringSplit/Tests/StringSplitUnitTest.cs

```diff
@@ -0,0 +1,188 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace Tests
+{
+    [TestClass]
+    public class StringSplitUnitTest
+    {
+        [TestMethod]
+        public void WhenSplitByNewLine_ThenCreateArrayOfSubstrings()
+        {
+            string highScoresText = $"15562{ Environment.NewLine }12988{ Environment.NewLine }9876{ Environment.NewLine }" +
+                $"6584{ Environment.NewLine }6292";
+
+            // Split the string by new line. The array will contain as many members as the number of lines in the text
+            string[] highScoresArray = highScoresText.Split(Environment.NewLine);
+
+            // Print each array member
+            for (int i = 0; i < highScoresArray.Length; i++)
+            {
+                Console.WriteLine($"High Score { i + 1 }: { highScoresArray[i] }");
+            }
+
+            //// Output:
+            ////High Score 1: 15562
+            ////High Score 2: 12988
+            ////High Score 3: 9876
+            ////High Score 4: 6584
+            ////High Score 5: 6292
+
+            Assert.IsTrue(highScoresArray.Length == 5);
+            Assert.IsTrue(highScoresArray[0] == "15562");
+            Assert.IsTrue(highScoresArray[1] == "12988");
+            Assert.IsTrue(highScoresArray[2] == "9876");
+            Assert.IsTrue(highScoresArray[3] == "6584");
+            Assert.IsTrue(highScoresArray[4] == "6292");
```

Comment:
This is excellent, but you need to use Assert.AreEqual instead of IsTrue. Apply to other examples.